recipes-core/softwarecontainer/files/io.qt.ApplicationManager.Application.json

```diff
@@ -0,0 +1,50 @@
+{
+  "version": "1",
+  "capabilities": [
+    {
+      "name": "io.qt.ApplicationManager.Application",
+      "gateways": [
+        {
+          "id": "dbus",
+          "config": [
+            {
+             "dbus-gateway-config-system": [
+                {
+                  "direction": "*",
+                  "interface": "*",
+                  "object-path": "*",
+                  "method": "*"
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "id": "wayland",
+          "config": [
+            {
+              "enabled": true
+            }
+          ]
+        },
+        {
+          "id": "devicenode",
+          "config": [
+            {
+              "name": "/dev/dri/card0"
+            },
+            {
+              "name": "/dev/dri/renderD128"
```

Comment:
Is this device needed or is this just an example? If it is an example I think it could be removed and only keep the tty stuff below.